R/utils.R

```diff
@@ -19,17 +19,28 @@
 #' )
 #' assay(se)
 #' assay(relativeAb(se))
+#' 
+#' utility function for relativeAb to return counts per million
 #'
+#' @description
+#' This function calculates the relative abundance of each feature in the SummarizedExperiment 
+#' object containing count data, expressed as counts per million (CPM) 
+#' 
+#' @returns se returns a new SummarizedExperiment object with counts per million
```

Comment:
Remove "se" from this text, since this describes the return value and not the parameter se

Reply:
Okay, Thank you for the corrections